zenodo/modules/theme/views.py

```diff
@@ -58,3 +59,14 @@ def sanitize_html(value):
 def current_user_is_admin():
     """Returns ``True`` if current user has the ``admin-access`` permission."""
     return Permission(ActionNeed('admin-access')).can()
+
+
+@blueprint.app_template_filter('date_record_limit')
```

Comment:
Instead of a filter you can also try a [Jinja test](https://jinja.palletsprojects.com/en/3.0.x/templates/#tests). Imagine a usage like (naming to be figured out :) ):

```jinja
{% if record is removable_record %}
```

or 

```jinja
{% if record.created is date_past(days=30) %}
```

Reply:
Thank you for the explanation and feedback! I have made a new commit with a test https://github.com/zenodo/zenodo/pull/2216/commits/9623218245aa9e7e46440979602c9e42db8c57ea